source/app/controllers/index.rb

```diff
@@ -1,3 +1,29 @@
 get '/' do
-  erb :index
+  redirect '/game'
 end
+
+get '/game' do
+ erb :new_game
+end
+
+post '/game' do
+
+  # p "Player 1 id:  #{params[:player_1_id]} , Player 2 id: #{params[:player_2_id]}, Player 1 token id: #{params[:player_1_token_id]}, Player 2 token id #{params[:player_2_token_id]} "
+
+  current_game_logic = GameLogic.new(params[:player_1_id], params[:player_2_id], params[:player_1_token_id], params[:player_2_token_id])
+
+  @winner_id, @loser_id, @winner_token_id, @loser_token_id = current_game_logic.outcome?
```

Comment:
this line really confuses me :sweat_smile: General rule of thumb, don't try to assign a bunch of values like this in one line. It becomes a little cryptic and hard to follow.